datm/cime_config/buildnml

```diff
@@ -180,6 +180,16 @@ def _create_namelists(case, confdir, inst_string, infile, nmlgen, data_list_path
 
     nmlgen.init_defaults(infile, config)
 
+    # Get anomaly forcing
+    # First, look in namelist
+    anomaly_forcing = nmlgen.get_value("anomaly_forcing")
+    if anomaly_forcing[0] is None:
```

Comment:
Maybe this should be 
`if anomaly_forcing and anomaly_forcing[0] is None:`

Reply:
I kept it mostly how it was in the original (although now it's moved and I deleted the `not`):
https://github.com/ESCOMP/CDEPS/blob/8b7314b95b9135ddaab31dc28e28baf52ea13841/datm/cime_config/buildnml#L208-L210

In testing, cases without `anomaly_forcing` in `user_nl_datm` work fine. Is there a different scenario you're worried about that I could test?

Reply:
If you are convinced that anomaly_forcing is never None at line 209 then I'm okay.

Reply:
It's never `None` as things are currently written, but I've incorporated your idea to improve robustness to future changes: b1f1751.